core/src/test/java/nl/nn/adapterframework/jdbc/dbms/SqlTranslatorTestOracleToH2.java

```diff
@@ -0,0 +1,28 @@
+package nl.nn.adapterframework.jdbc.dbms;
+
+import java.sql.SQLException;
+
+import org.junit.Ignore;
+
+import nl.nn.adapterframework.jdbc.JdbcException;
+import nl.nn.adapterframework.jdbc.QueryExecutionContext;
+
+public class SqlTranslatorTestOracleToH2 extends OracleToH2TranslatorTest {
+
+	@Override
+	protected String convertQuery(QueryExecutionContext queryExecutionContext, boolean canModifyQueryExecutionContext) throws JdbcException, SQLException {
+		SqlTranslator translator = new SqlTranslator("Oracle", "H2");
+		return translator.translate(queryExecutionContext.getQuery());
+	}
+
+	@Ignore("too hard for SqlTranslator to create identity column")
+	@Override
+	public void testIgnoreAlterTableIbisStore() throws JdbcException, SQLException {
+		//super.testIgnoreAlterTableIbisStore();
+	}
+	@Ignore("too hard for SqlTranslator to create identity column")
+	@Override
+	public void testConvertQueryCreateTableIbisStore() throws JdbcException, SQLException {
+		//super.testConvertQueryCreateTableIbisStore();
+	}
```

Comment:
De IbisStore is wel de belangrijkste tabel. Omdat dit nu uit staat wil dat zeggen dat geen `identity` translaties gedaan kunnen worden?